supplier_app/tests/test_view.py

```diff
@@ -1031,14 +1030,13 @@ def test_change_taxpayer_status_to_active_sends_email_notification(self):
                 kwargs=self.kwargs
             )
         )
-
         self.assertEqual(
             mail.outbox[0].subject,
             email_notifications['taxpayer_approval']['subject']
         )
         self.assertIn(
             settings.SUPPLIER_HOME_URL,
-            mail.outbox[0].body
+            mail.outbox[0].alternatives[0][0]
```

Comment:
Just to know, what are the alternatives?

Reply:
in this case in send_email_notification method we are using strip_tags for plain text if html does not render this makes that the body does not show html tags and can't assert the href inside the achor tag.The alternatives contains the whole html files.